### PR TITLE
Feature/structurier demo

### PR DESCRIPTION
**Pull request by PeterBridgerMadeTech**
Updated the visualising architecture path with references to the Structurizr CoP demo

### CODE REVIEW

File: learning_paths/visualising_architecture/3_structurizr.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# Structurizr
+We recommend using the [C4 Model](https://c4model.com/) in conjunction with the [Structurizr DSL](https://github.com/structurizr/dsl) as a way to automatically generate the C4 diagrams from code. This allows for the Structurizr code to be kept in the same repository as the code of the system, making it easier to keep the two up to date together.
+
+A [Structurizr Lite quick start template](https://github.com/madetech/structurizr-template) has been created that you can use as a starting point in your own projects. 
+
+## Quick overview
+All the code to define the model, relationships and views are held in a single `workspace.dsl` file that Structurizr uses to build the visual diagams.
+![Structurizr workspace](/images/structurizr_workspace.jpg)
```
Comment (TimHacker) [inline learning_paths/visualising_architecture/3_structurizr.md:8]: In the diff previous this image is showing on the same line as the preceding sentence? Will the extra line help?

```suggestion

![Structurizr workspace](/images/structurizr_workspace.jpg)
```

**Review by TimHacker — APPROVED** [review]
Nice! Is it worth including a note somewhere that all the COP talks are internal only as this is open source.

**Review by smckee-mt — APPROVED** [review]
Looks good, nice one Peter!

File: learning_paths/visualising_architecture/3_structurizr.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+## Watch
+### Structurizr: Software architecture models as code CoP session (1st August 2023)
+[![Software architecture models as code](/images/structurizr.jpg)](https://drive.google.com/file/d/1mJ2TpHPmiwDel0GhoIQdVv87cXfTwLCf/view)
+**Made Tech internal only video**
+
+Peter Bridger explains how to get up and running with Structurizr to create your own interactive diagrams, along with using the documentation and ADR abilities of Structurizr Lite using the quick start template mentioned previously.
+
+- [Recording](https://drive.google.com/file/d/1mJ2TpHPmiwDel0GhoIQdVv87cXfTwLCf/view)
+- [Slides](https://docs.google.com/presentation/d/1WDT0kEZoicyoUUp96QXznWwGv8lmvfAno0s_irAkXX0/edit#slide=id.g25db1f62abd_0_0)
```
Comment (TimHacker) [inline learning_paths/visualising_architecture/3_structurizr.md:36]: This is jumping to one of the later slides. Is it worth removing that?

```suggestion
- [Slides](https://docs.google.com/presentation/d/1WDT0kEZoicyoUUp96QXznWwGv8lmvfAno0s_irAkXX0/edit)
```

**Pull request merged**